frontend/src/timeline.module.css

```diff
@@ -0,0 +1,254 @@
+        font-family: 'Inter';
+    }
+    
+    .arrowText1::after {
+        content: '↑';
+    }
+    
+    .arrowText2::after {
+        content: '↓';
+    }
+
+    .weekdayText {
+        display: none;
+    }
+
+    .mobileTimelineContainer {
+        height: 100vh;
+        width: 19.2vw;
+    }
+    
+}
+
+@media screen and (min-width: 570px) {
+    body {
+        padding: 0;
+        margin: 0;
+        font-family: 'Inter';
+    }
+
+    .timelinePageContainer {
+        color: black;
+        text-align: center;
+    }
+
+    .PCTimelineContainer {
+        color: black;
+        text-align: center;
+    }
+
+    .timelineBarContainer {
+        z-index: 1;
+        padding: 0;
+        margin: 0;
+        display: flex;
+        max-width: 100vw;
+        align-content: center;
+        background-color: pink;
+    }
+
+    .daysContainer {
+        width: 84vw;
+        height: 9vh;
+        border-top: 0.4vh solid gray;
+        border-bottom: 0.4vh solid gray;
+        display: flex;
+    }
+
+    .arrowContainer {
+        width: 8vw;
+        height: 9vh;
+        font-size: 2.34vw;
+        vertical-align: center;
+        line-height: 8.4vh;
+    }
+
+    .weekdayInitial {
+        display: none;
+        padding: 0;
+        margin: 0;
+    }
+
+    .arrowText1::after {
+        content: '←';
+        padding: 0;
+        margin: 0;
+    }
+
+    .arrowText1 {
+        padding: 0;
+        margin: 0;
+        margin-right: 20%;
+        text-align: right;
+    }
+
+    .arrowText2::after {
+        content: '→';
+        padding: 0;
+        margin: 0;
+    }
+
+    .arrowText2 {
+        padding: 0;
+        margin: 0;
+        margin-left: 20%;
+        text-align: left;
+    }
+
+    .dayContainerBox {
+        padding: 0;
+        margin: 0;
+        width: 14.3%;
+    }
+
+    .weekdayText {
+        padding: 0;
+        margin: 0;
+        margin-top: 8.5%;
+        font-weight: 600;
+        font-size: 1.95vh;
+    }
+
+    .dateText {
+        padding: 0;
+        margin: 0;
+        margin-top: 1%;
+        font-size: 1.37vh;
+    }
+
+    .weekTitle {
+        font-size: 3.1vh;
+        margin: 0;
+        margin-bottom: 0.7vh;
+        padding: 0; 
+        font-weight: 600;
+    }
+
+    .dateRange {
+        font-size: 2.3vh;
+        margin: 0;
+        padding: 0;
+        font-weight: 500;
+    }
+
+    .mobileTimelineFrameContainer {
+        width: 565px;
+        height: 1108px;
+        text-align: center;
+        margin: 0 auto;
+        
+        .mobileTimelineContainer {
+            height: 100vh;
+            width: 19.2%;
+            text-align: center;
+            color: black;
+        }
+
+        .timelineTitle {
+            text-align: center;
+        }
+
+        .timelineBarContainer {
+            z-index: 1;
+            padding: 0;
+            margin: 0;
+            display: flex;
+            height: 100%;
+            align-content: center;
+            background-color: #D9D9D9;
```

Comment:
please switch to `hsl` equivalent for consistency with project - hover over it in vscode and click on the hex above the colour swatch a couple times to convert it